spec/factories/planning_application.rb

```diff
@@ -8,6 +8,8 @@
     status           { :in_assessment }
     in_assessment_at { Time.zone.now }
     ward             { Faker::Address.city }
+    documents_validated_at { Time.zone.today }
```

Comment:
this one doesn't need to be part of the exposed api field too?

Reply:
that's a good question - I don't think it's specified at the moment, so I didn't add it, but I need to check!